README.md

```diff
@@ -1 +1,34 @@
-# Capstone-proj
\ No newline at end of file
+# Capstone-proj 
+ 
+My capstone project, built with HTML, CSS and JAVASCRIPT to dominstrate all what I have learned so far from microverse. I did this with all pleasure.
+
+# Build with 
+ - HTML 
+ - CSS
+ - JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://sirriryisa.github.io/Portolio_mobile_view/#)
```

Comment:
Please make sure to change the live demo link, as suggested. You can deploy your project using [neftlify](https://www.netlify.com/) or using [github page](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) .